src/Samples/SarifToCsv/CsvWriter.cs

```diff
@@ -101,6 +102,11 @@ public void Write(int value)
             Write(value.ToString());
```

Comment:
>value [](http://example.com/codeflow?start=18&length=5)

Please use

value.ToString(CultureInfo.InvariantCulture)

here. Also, please review any other numeric values written in this code to scrub for the same problem.